q2_american_gut/assets/report/index.html

```diff
@@ -0,0 +1,10 @@
+{% extends "base.html" %}
+
+{% block head %}
+    <title>American Gut Report</title>
+{% endblock %}
+
+{% block content %}
+        <h1>American Gut Report for {{ name }}</h1>
+        <img src='./resources/ag_logo.png' alt='American Gut Logo'>
```

Comment:
Open question, in the skeleton, should we accommodate other projects and abstract the branding out (e.g., British Gut)?